### PR TITLE
fix(docs): Remove issue id from comments.

### PR DESCRIPTION
**Pull request by jaideepghosh**
## Description:
In the doc (https://angular.io/api/core/ViewChild#usage-notes) example contains a line of unnecessary comment. In that same way, multiple doc file contains the same. So removing them to improve the readability of the doc.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/benchpress/src/firefox_extension/lib/main.ts [hunk 1]
```diff
@@ -12,9 +12,9 @@ const ParserUtil = require('./parser_util');
 
 class Profiler {
   private _profiler: any;
-  // TODO(issue/24571): remove '!'.
+  
   private _markerEvents !: any[];
-  // TODO(issue/24571): remove '!'.
```
Comment (santoshyadav198613) [inline packages/benchpress/src/firefox_extension/lib/main.ts]: I can see this issue is still open, let's wait for confirmation before we remove this.
https://github.com/angular/angular/issues/24571

Comment (jaideepghosh) [reply packages/benchpress/src/firefox_extension/lib/main.ts]: Okay, but showing a comment in official doc's examples is not recommended. That's why I've created this PR, to remove the comments.

Comment (santoshyadav198613) [reply packages/benchpress/src/firefox_extension/lib/main.ts]: Yes agreed, but the purpose here is to update the examples as well when the issue gets resolved, so we should not miss it.

Comment (jaideepghosh) [reply packages/benchpress/src/firefox_extension/lib/main.ts]: Agreed, but issue is already created, so once it is resolved, we can work on the required fixes.
Having an issue id in every file is not making the code readable. And specially if it is something related with official docs, then we must focus on it.

**Review by Splaktar — CHANGES_REQUESTED** [review]
This is related to https://github.com/angular/angular/issues/24571.

https://github.com/angular/angular/issues/24571#issuecomment-404606595 provides guidance on when/if/how the comments and the `!` should be removed. We cannot simply remove all of the comments without resolving whether or not the `!` should remain or the code should be refactored in some other way.

**Comment by brandonroberts** [comment]
@jaideepghosh thanks for the PR. This should only apply to files under the `packages/examples` folder. The remaining ones in the codebase are to be left untouched.

**Comment by kapunahelewong** [comment]
@jaideepghosh if you need help with anything, please tag me. Thanks!

**Comment by petebacondarwin** [comment]
Closing this one as it is not as simple as deleting the comments. We need to address the reason for the `!` in each case and fix appropriately. There should be separate PRs for each block of changes.

**Pull request closed without merging**